open-apparel-registry/open-apparel-registry
# Remove "show other contributor info" embed option and hide other contributor information in embed mode

## Overview

Add a migration to remove the "show other contributor info" option from the embed config model, and remove related code from the back-end. The front-end control for this option is also removed.

Hide other facility locations, names, addresses, and contributors on the facility details panel in embed mode.

Connects #1258 

## Demo

| Embed mode  | Main app |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1042475/118718866-c1217d00-b7f5-11eb-9bec-07363a427df5.png) | ![image](https://user-images.githubusercontent.com/1042475/118718904-cc74a880-b7f5-11eb-9a0e-521a66ade9b0.png) |


## Testing Instructions

- Run `./scripts/update` to apply the migration
- Visit Settings > Embed
- Scroll down to the "Include these fields" section, and verify the "Information from other contributors" checkbox is gone.
- Scroll to the map, and view a facility that has multiple other locations/names/addresses/contributors. Ensure none of those fields are displayed.
- Visit the main app, and view the same facility. Ensure any other locations/names/addresses/contributors are displayed.

## Checklist
